### PR TITLE
Dockerize devito application

### PR DESCRIPTION
**Pull request by c-w**
Paired with @navjotk

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,12 +31,17 @@ provided at the download links. You will need the Python 3.6 version.
 
 To install Devito, including examples, tests and tutorial notebooks, 
 follow these simple passes:
-```
+
+```sh
+# get the code
 git clone https://github.com/opesci/devito.git
 cd devito
-conda env create -f environment.yml
```
Comment (FabioLuporini) [inline README.md]: I think both options should stay. Conda, in particular, should be the way to go for developers.

File: README.md [hunk 2]
```diff
@@ -31,12 +31,17 @@ provided at the download links. You will need the Python 3.6 version.
...
-source activate devito
-pip install -e .
+
+# run tests
+docker-compose run devito /tests
```
Comment (FabioLuporini) [inline README.md]: this assumes the presence of docker

File: README.md (hunk 1 above)
Comment (mloubout) [reply README.md]: Would keep conda too.

Comment (c-w) [reply README.md]: Sounds good. I've restored the previous README state.

File: README.md (hunk 2 above)
Comment (c-w) [reply README.md]: Added a link to the Docker documentation.

**Comment by FabioLuporini** [comment]
People are confirming that this does work as expected. @navjotk @mloubout @ggorman any other comments?

**Comment by navjotk** [comment]
I like it. Waiting for @mloubout to update his review.

**Comment by FabioLuporini** [comment]
Cool, merging.

**Comment by FabioLuporini** [comment]
Merged.

**Pull request merged**